packages/hooks/src/useP2POrderList.ts

```diff
@@ -0,0 +1,47 @@
+import { usePaginatedFetch, useSubscription } from '@deriv/api';
+import { useEffect, useState } from 'react';
+
+type TOrder = NonNullable<
+    NonNullable<ReturnType<typeof useSubscription<'p2p_order_list'>>['data']>['p2p_order_list']
+>['list'][number];
+
+/** This custom hook returns the list of active orders by calling 'p2p_order_list' endpoint */
+const useP2POrderList = (
+    payload: NonNullable<Parameters<typeof usePaginatedFetch<'p2p_order_list'>>[1]>['payload']
+) => {
+    const { data, ...rest } = usePaginatedFetch('p2p_order_list', { payload });
+    const { data: subscription_data, subscribe, ...subscription_rest } = useSubscription('p2p_order_list');
```

Comment:
<img width="424" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/19817262-0d94-4ce3-badd-38b236c3a973">

<img width="424" src="https://media.tenor.com/_BiwWBWhYucAAAAd/what-huh.gif">

Reply:
I am using the data from usepaginated fetch hook.That is why I am not using from subscription_data.Let me check that